src/measure_theory/measurable_space_def.lean

```diff
@@ -313,6 +313,10 @@ assume t (ht : generate_measurable s t), ht.rec_on h
   (assume s _ hs, measurable_set_compl m s hs)
   (assume f _ hf, measurable_set_Union m f hf)
 
+lemma generate_from_measurable_set' (m : measurable_space α) :
+  generate_from m.measurable_set' = m :=
```

Comment:
Is this the same as
```suggestion
  generate_from measurable_set[m] = m :=
```
I think @JasonKYi and I settled on avoiding `measurable_set'` in any lemmas with a notation something like this.

Reply:
I think I got confused a moment... that new lemma is the same as the existing `generate_from_measurable_set`. I'll remove it.

Reply:
But no, it's not exactly the same. The two lemmas are:
```
@[simp] lemma generate_from_measurable_set [measurable_space α] :
  generate_from {s : set α | measurable_set s} = ‹_› :=
le_antisymm (generate_from_le $ λ _, id) $ λ s, measurable_set_generate_from

lemma generate_from_measurable_set' (m : measurable_space α) :
  generate_from m.measurable_set' = m :=
generate_from_measurable_set
```
They are very close, but I did not manage to prove `comap_eq_generate_from` by using the first one.

Reply:
Here I used `measurable_set'` intentionally, because I want to get the set that was specified in the field `measurable_set'` when defining `comap`.

Reply:
@eric-wieser it is a bit different in this case since we want the set of all measurable sets rather than the measurability predicate. So `measurable_set[m]` (if it works) is actually defeq abuse in this case

Reply:
With some `convert` use, I managed to prove what I wanted with `generate_from_measurable_set` (see `comap_eq_generate_from`) and I removed `generate_from_measurable_set'`.